src/components/PlaylistCard.js

```diff
@@ -13,6 +14,22 @@ export default function PlaylistCard({ playlistObj }) {
       <Link href={`/playlists/edit/${playlistObj.id}`} passHref>
         Edit
       </Link>
+      <div className="col">
+        <Dropdown>
+          <Dropdown.Toggle variant="success" id="dropdown-basic">
+            ...
+          </Dropdown.Toggle>
+
+          <Dropdown.Menu>
+            <Dropdown.Item href="">
+              <Link href={`/playlists/${playlistObj.id}`} passHref>
+                Go to playlist
+              </Link>
+            </Dropdown.Item>
+            <Dropdown.Item href="">Add to playlist</Dropdown.Item>
```

Comment:
This section needs to be changed. I think it's from the song card. But we do not need to add a playlist to a playlist. 
Other than this I don't see any issues. 
Are we able to add a song to a playlist yet?

Reply:
Not yet.  Once I receive a second approval, I'll merge my, pull down from main, and start working on the form.